### ISSUE TITLE
[QUESTION] 

### ISSUE THREAD

**Issue by IshanGrover2004** [CCExtractor/ccextractor#1606]
CCExtractor version: 0.94

# In raising this issue, I confirm the following:

- [x] I have read and understood the [contributors guide](https://github.com/CCExtractor/ccextractor/blob/master/.github/CONTRIBUTING.md).
- [x] I have checked that the bug-fix I am reporting can be replicated, or that the feature I am suggesting isn't already present.
- [x] I have checked that the issue I'm posting isn't already reported.
- [x] I have checked that the issue I'm porting isn't already solved and no duplicates exist in [closed issues](https://github.com/CCExtractor/ccextractor/issues?q=is%3Aissue+is%3Aclosed) and in [opened issues](https://github.com/CCExtractor/ccextractor/issues)
- [x] I have checked the pull requests tab for existing solutions/implementations to my issue/suggestion.
- [x] I have used the latest available version of CCExtractor to verify this issue exists.
- [x] I have ticked all the boxes in this section and to prove it I'm deleting the section completely to remove boilerplate text.

# Necessary information

- Is this a regression (i.e. did it work before)? NO
- What platform did you use? LINUX
- What were the used arguments? Doesn't matter in this case

# Additional information

I was going through the codebase and testing things
I observed a function [`get_write_interval`](https://github.com/CCExtractor/ccextractor/blob/3f504412f54dc88cdcfbdbf56e4657d51a6d9b49/src/rust/src/decoder/tv_screen.rs#L106)
where this function returns the bounds in which captions are present. So that means if caption/dtvcc_symbol in that case is default or 0 then it should return `(0,0)` answer
But it is returning `(0,209)` 

Reason behind it:
`0x00.is_set()` is true

Question:
Is this a intentional behavior or just a mistake which never noticed?

**Comment by IshanGrover2004** [comment]
@PunitLodha @prateekmedia @cfsmp3 @canihavesomecoffee Do you think, is this a actual problem?

**Comment by cfsmp3** [comment]
@IshanGrover2004 You've been looking at the code more recently than we have :-)

Being blunt, in order to answer that I'd have to reread the specs, because while I remember the basics I definitely don't remember the details (or the coding decisions back then).

If it's broken, please fx it - ultimately all we care about is correctness in output.

**Comment by IshanGrover2004** [comment]
@cfsmp3 I guess this is not reflecting any of the output bcs it fills basically 0 value to array
But while doing something like `arr.is_empty()` returns false bcs it has 0 values but technically it should be true if its an actuall issue

**Comment by IshanGrover2004** [comment]
Yes @cfsmp3, It was a minute mistake which I corrected in PR #1609
